src/librustdoc/clean/mod.rs

```diff
@@ -1211,56 +1211,47 @@ pub(crate) fn clean_middle_assoc_item<'tcx>(
                     tcx.generics_of(assoc_item.def_id),
                     ty::GenericPredicates { parent: None, predicates },
                 );
-                // Move bounds that are (likely) directly attached to the associated type
```

Comment:
Ah, you replaced the comment I updated with its older version ^^' No worries, I'm gonna revert that as part of a bigger PR in the future.